test.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# Usage: ./test.sh <make target>
+# Execute a makefile target inside a Docker environment
+
+MAKE_TARGET="test-coverage-all-codeclimate"
+if [[ -n "$1" ]]; then
+  MAKE_TARGET="$1"
+fi
+
+docker stop commander-int-ssh-server || true
+docker rm commander-int-ssh-server || true
+docker stop commander-integration-go-test || true
+docker rm commander-integration-go-test || true
+docker network rm commander_test || true
+
+docker build -t commander-int-ssh-server -f integration/containers/ssh/Dockerfile .
+docker network create  --driver=bridge --subnet=172.28.0.0/16 commander_test
+
+docker run -d \
+  --ip=172.28.0.2 \
+  --network commander_test \
+  --name int-ssh-server \
```

Comment:
I think you accidentally added two names here.

Reply:
Good catch, thanks!